scripts/setup_hooks.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+git config core.hooksPath "./hooks"
```

Comment:
(To clarify, the question is what the default behavior for this hook, which does linting and runs tests before pushing to github, should be. Should it be enabled automatically in the default workflow, or be more of an optional add-on?)